### ISSUE TITLE
[Accessibility] Toast notifications not announced by screen readers

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#12528]
### Steps To Reproduce

1. Run a screen reader
2. Perform an action that triggers a toast notification (e.g. trying to unlock a vault with the wrong password)

### Expected Result

Toast notification should be announced by the screen reader (using a live region notification).

### Actual Result

Toast notification is not announced at all by screen readers.

### Screenshots or Videos

https://github.com/user-attachments/assets/5eedb93b-4ee3-44f6-a5f4-9b1b0964cc04

Video: Chrome/NVDA, trying to unlock a vault with an empty password - a toast error notification is shown, but NVDA remains fully silent.

### Additional Context

Possibly still related to https://github.com/bitwarden/clients/issues/3487

I also filed an issue ages ago upstream https://github.com/scttcper/ngx-toastr/issues/957 but not sure if anything came from that.

This may be interesting/relevant: https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/

### Operating System

Windows

### Operating System Version

11

### Web Browser

Chrome

### Browser Version

131.0.6778.205 (Official Build) (64-bit)

### Build Version

2024.12.2

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SergeantConfused** [comment]
Hello @patrickhlauke,

Thank you for your report. I have flagged this to the Engineering department; please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you again,

**Comment by danielleflinn** [comment]
Closing as duplicate of https://github.com/bitwarden/clients/issues/3487 as the team will be fixing the issue holistically across clients where this component is used.
